core/connector/src/main/java/org/exoplatform/ecm/connector/dlp/FileDlpConnector.java

```diff
@@ -167,6 +168,7 @@ protected void treatItem(String entityId, Collection<SearchResult> searchResults
       RestoredDlpItem restoredDlpItem = findRestoredDlpItem(node.getUUID());
       if (!node.getPath().startsWith("/"+DLP_SECURITY_FOLDER+"/") && (restoredDlpItem == null ||  getNodeLastModifiedDate(node) > restoredDlpItem.getDetectionDate())) {
         workspace.move(node.getPath(), "/" + DLP_SECURITY_FOLDER + "/" + fileName);
+        Utils.removeDeadSymlinks(node);
```

Comment:
I'm not sure that we should remove symlink. 
When we keep the symlink, and when the target document goes in quarantine, what is the behaviour when you click on the symlink ?

Reply:
I have discussed with Sameh and this the normal behavior is to remove symlinks when the document is moved to the quarantine. The symlink is not accessible when the document is moved to the quarantine page.

Reply:
Why ? When the document is restored, the symlinks are not restored. 
Can you try to answer to this  :
When we dont remove symlink,  what is the behaviour when you click on a symlink of a document which is in quarantine ?